addons/web/static/src/js/widgets/data_export.js

```diff
@@ -441,14 +441,16 @@ var DataExport = Dialog.extend({
         var export_format = this.$export_format_inputs.filter(':checked').val();
 
         framework.blockUI();
+        var context = pyeval.eval('contexts', [this.dataset._model.context()]);
+        context.active_test = false,
```

Comment:
I've got no opinion on the necessity of active_test, but pyeval.eval('contexts') takes multiple contexts and merges them together so this could be `pyeval.eval('contexts', [this.dataset._model.context(), {active_test: false}]);`